results.pm

```diff
@@ -439,4 +443,31 @@ sub writelog
 	close($fh);
 }
 
+sub openqa_formatter
+{
+	my ($log_data) = @_;
+	my $json_data = {
+		'environment' => $log_data->{sysinfo},
+		'stats' => $log_data->{tests}->{stats},
+		'results' => []
+	};
+
+	for my $r (@{$log_data->{tests}->{results}}) {
+		my $result = 'unk';
+		$result = 'pass' if ($r->{passed} || $r->{skipped});
+		$result = 'fail' if ($r->{failed} || $r->{broken} || $r->{warnings});
+		my $t = {
+		'test_fqn' => $r->{tid},
+		'status' => $result,
+		'test' => {
+			'duration' => $r->{runtime},
+			'result' => $result,
+			'log' => join($/, @{$r->{log}})
+		}
+	};
+	push(@{$json_data->{results}}, $t);
+	}
```

Comment:
The indentation around here is messed up.

Reply:
please see #27  - this will save your time from writing such messages :)